cli/js/web/url.ts

```diff
@@ -214,7 +214,6 @@ function resolvePathFromBase(
 
 function isValidPort(value: string): boolean {
   // https://url.spec.whatwg.org/#port-state
-  if (value === "") true;
```

Comment:
Yikes. Good catch.

Reply:
This is intentional and required... it represents the no-port case for the URL constructor.

Reply:
Oh oops, I see the error.

Please change it to
```js
if (value === "") return true;
```